text/0000-stack-metadata.md

```diff
@@ -0,0 +1,95 @@
+# Meta
+[meta]: #meta
+- Name: Additional Stack Metadata
+- Start Date: 2020-05-12
+- Author(s): kvedurmu
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+We currently expose the Stack ID and Mixins via Labels on the build/run images. It would be great if we could leverage Labels to allow stack authors to specify additional image metadata (e.g OS Distro, Maintainer, Stack Version, etc.). The purpose of this RFC is to enumerate on additional metadata we want to expose in our stack images and standardize on these labels.
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this?  
+Exposing additional metadata in stack images will make it easier for end users to debug stack related issues. Also, by exposing this metadata, CNB platforms such as kpack can leverage this information to improve their UX. 
```

Comment:
What would `kpack` or a platform do with this data?

Reply:
`kpack` provides a `stack` resource that currently only reports the digest of the provided run/build image. It would be useful to report more detailed metadata status of that stack. That way users could better understand where the stack comes form. Something like:

```
Id:             io.buildpacks.stacks.alpine
Run Image:      alpine/run@sha256:d70bf0fe11d84277997c4a7da94b2867a90d6c0f55add4e19b7c565d5087206f
Build Image:    alpine/build@sha256:663a1781cb60a3feb43b9a2a2eae6681ca0124509b47719b5a6bf770c7638712
Mixins:         ca-certificates, libssl1.1
Url:            https://github.com/alpinelinux/cnb-alpine
ReleaseDate:    2020-05-12T05:17:02.390472
....
```

Reply:
We could even expose some of this output in existing `pack` commands such as `rebase` if we wanted to.

Reply:
It would be nice to include this info in the `pack inspect-image` output as well

Reply:
Can we include the kpack or pack examples in the RFC? :)

Reply:
@hone Do the latest changes resolve this comment?

Reply:
yep!